### PR TITLE
Follow-up on negotiation

### PR DESCRIPTION
**Pull request by D4nte**
Implement TODOs/comments from comit-network/comit-js-sdk#60 & #128.

### CODE REVIEW

File: new_project/examples/separate_apps/src/taker.ts [hunk 1]
```diff
@@ -22,14 +24,27 @@ import { checkEnvFile, startClient } from "./lib";
...
     const makerClient = new MakerClient("http://localhost:2318/");
 
     // take an order from a maker
     // Accept any order
-    const isOrderAcceptable = () => true;
-    const { order, swap } = await takerNegotiator.negotiateAndSendRequest(
-        taker.comitClient,
+    const isOrderAcceptable = (order: Order) => {
+        if (order.ask.asset !== "ether" || order.bid.asset !== "bitcoin") {
+            // These are'nt the droids you're looking for
```
Comment (bonomat) [inline new_project/examples/separate_apps/src/taker.ts]: 🤖🦾

**Comment by D4nte** [comment]
bors cancel

**Pull request merged**